app/src/main/res/values/strings.xml

```diff
@@ -306,60 +306,60 @@
         9. Other memory and crash fixes.
     </string>
 
-    <string name="connection_http3">HTTP3</string>
+    <string name="connection_http3">HTTP/3</string>
 
     <!-- Settings Fragment strings -->
     <string name="settings_ip_network_heading">Choose IP version</string>
     <string name="settings_ip_network_desc">Choose between Internet Protocol versions.</string>
 
     <string name="settings_ptrans_heading">Force IPv4 egress</string>
-    <string name="settings_ptrans_desc">Translate outbound IPv6 traffic to IPv4 on-device. Enable if you are connected to an IPv4-only network.</string>
+    <string name="settings_ptrans_desc">Translate outbound IPv6 traffic to IPv4 on-device. Use this if you are connected to an IPv4-only network.</string>
```

Comment:
```suggestion
    <string name="settings_ptrans_desc">Translate outbound IPv6 traffic to IPv4 on-device. Useful for IPv4-only networks.</string>
```

Reply:
It isn't "useful"... in fact, it is "best effort" (which is why it isn't enabled / turned-on by default). The current wording which goes `Use this if...` is appropriate in this case.